### PR TITLE
[Jetsurvey] Photo was not showing with rememberCoilPainter

### PR DESCRIPTION
**Pull request by angeles-bilbao6**
Preview image stop showing after migrating to rememberCoilPainter

https://github.com/android/compose-samples/issues/460

Taken photo should be clipped, keep size of photo frame the same

Before
image

After
image

### CODE REVIEW

**Comment by chrisbanes** [comment]
So the reason this is happening is because the main UI is in an `LazyColumn`, so each item implicitly has no max height. This means that using `fillMaxHeight()` has no effect (as there is no max size) and results in the item having a height of 0. 

`rememberCoilPainter()` sees this size and skips loading due to the 0 height. Working on a fix now

**Comment by JoseAlcerreca** [comment]
@angeles-bilbao6 please rebase this PR to include the fix in #478

**Comment by angeles-bilbao6** [comment]
@JoseAlcerreca Done, I kept the max height value because else way the photo won't respect the design expected height.

**Comment by chrisbanes** [comment]
Hmmm, where did the value of `302.dp` come from? I don't think this is right, and will only 'work' for specific display sizes. If we want to force an aspect ratio, we can use `Modifier.apsectRatio(...)`.

**Comment by angeles-bilbao6** [comment]
@chrisbanes You are right, I just made the change

File: Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt [hunk 1]
```diff
@@ -346,8 +347,8 @@ private fun PhotoQuestion(
                     contentDescription = null,
                     modifier = Modifier
                         .fillMaxWidth()
-                        // We set a minimum height, so that Coil has a height to initially use
-                        .heightIn(min = 96.dp),
+                        .heightIn(96.dp)
+                        .aspectRatio(1.22f)
```
Comment (chrisbanes) [inline Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt:351]: `1.22f` is an interesting aspect ratio. I'd probably use `aspectRatio(4 / 3f)` which is more typical and common.

Comment (angeles-bilbao6) [reply Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt:351]: Figma design has an image size of 371 / 302 px that is why I chose that aspect ratio, but I will update to use 4/3f

**Review by chrisbanes — APPROVED** [review]

**Comment by JoseAlcerreca** [comment]
Thanks!

**Pull request merged**